gap/matrix/slconstr.gi

```diff
@@ -2465,7 +2471,7 @@ SLPforElementFuncsMatrix.SLConstructive := function(ri,x)
 end;
 
 #! @BeginChunk NaturalSL
-#! TODO
+#! TODO: This function is not used at the moment, but SLCR.FindHom is used.
```

Comment:
Iissue #257 would also take care of this comment

Reply:
I'm removing this comment and will do #257 and #258 next.